<!-- LAUNCH FOR EXECUTING IN DEVELOPMENT COMPUTER -->

<launch>
  <rosparam command="load" file="$(find coordination_formation_control_pkg)/config/missionConfig.yaml" />
  <rosparam command="load" file="$(find coordination_formation_control_pkg)/config/formationConfig.yaml" />

  <node name="mission_controller" pkg="coordination_formation_control_pkg" type="mission_controller" respawn="false"
    output="screen"/>



        <group ns="omniwheel_1">
             <node name="omniwheel_node" pkg="coordination_formation_control_pkg" type="omniwheel_node" output="screen" args="1" launch-prefix="xterm -e">
             </node>
         </group>
         <group ns="omniwheel_2">
              <node name="omniwheel_node" pkg="coordination_formation_control_pkg" type="omniwheel_node" output="screen" args="2" launch-prefix="xterm -e">
              </node>
          </group>
  <!-- Initialize all the drones first -->
  <group ns="uav_1">
       <node name="drone_node" pkg="coordination_formation_control_pkg" type="drone_node" output="screen" args="1" launch-prefix="xterm -e">
       </node>
   </group>

   <group ns="uav_2">
        <node name="drone_node" pkg="coordination_formation_control_pkg" type="drone_node" output="screen" args="2" launch-prefix="xterm -e">
        </node>
    </group>
    <group ns="uav_3">
         <node name="drone_node" pkg="coordination_formation_control_pkg" type="drone_node" output="screen" args="3" launch-prefix="xterm -e">
         </node>
     </group>


     <!-- <group ns="uav_4">
          <node name="drone_node" pkg="coordination_formation_control_pkg" type="drone_node" output="screen" args="4" launch-prefix="xterm -e">
          </node>
      </group> -->











<!-- launch dynamic reconfigure -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" respawn="false"
    output="screen" launch-prefix="xterm -e" />

</launch>
